src/app/hearing/services/possible-actions-service.ts

```diff
@@ -0,0 +1,92 @@
+import { Injectable } from '@angular/core';
+import { MatDialog } from '@angular/material';
+import { HearingActions } from '../models/hearing-actions';
+import { DialogWithActionsComponent } from '../../features/notification/components/dialog-with-actions/dialog-with-actions.component';
+import { DEFAULT_DIALOG_CONFIG } from '../../features/transactions/models/default-dialog-confg';
+import { HearingService } from './hearing.service';
+import { TransactionDialogComponent } from '../../features/transactions/components/transaction-dialog/transaction-dialog.component';
+import { Hearing } from '../models/hearing';
+
+@Injectable()
+export class PossibleActionsService {
+
+    possibleActions = {
```

Comment:
what type is it? Have you got a model for it? It looks like a foundation for extension later on